backend/src/main/java/com/benine/backend/database/MySQLDatabase.java

```diff
@@ -132,7 +121,6 @@ public void updatePreset(Preset preset) throws SQLException {
         list.add(getPresetsFromResultSet(resultset));
       }
       resultset.close();
-      statement.close();
     } finally {
       if (statement != null) {
```

Comment:
I see this piece of code is returning often. Maybe there could be a close(statement) method, which also does the null check.